### ISSUE TITLE
Installing jmvtools

### ISSUE THREAD

**Issue by karimhbadr1** [jamovi/jmvtools#7]
Hello!

I've been trying to install "jmvtools" and been following the instructions on the webpage here: https://dev.jamovi.org/tuts0101-getting-started.html 

I ran this bit of code:
`install.packages('jmvtools', repos=c('https://repo.jamovi.org', 'https://cran.r-project.org'))`

I couldn't find the package on cran so RStudio went with looking at it on Jamovi repo.

I get the following error:
![jmvtools](https://github.com/jamovi/jmvtools/assets/93216666/e0b15f78-c961-4b1d-88de-9ed97f092dab)

It seems on the repo that this is under 3.3 version of R, unlike my current R which is 4.3 so I manually went to : https://repo.jamovi.org/src/contrib/
And tried to download https://repo.jamovi.org/src/contrib/jmvtools_2.4.6.tar.gz and https://repo.jamovi.org/src/contrib/node_1.2.tar.gz but got the error upon downloading that "File wasn't available on site". For reference, I am using Google Chrome.

Another approach I tried is going to https://repo.jamovi.org/bin/windows/contrib/3.3/ and install https://repo.jamovi.org/bin/windows/contrib/3.3/node_1.0.zip but still got the same error "File wasn't available of site".

Am I doing something wrong here? Appreciate the help!

**Comment by jonathon-love** [comment]
hi,

the error is 'Failure when receiving data from the peer' which is R's way of saying that the download did not complete successfully. that this happened twice in a row suggests your internet connection is possibly a bit spotty.

try again, and you should find it all works.

cheers

**Comment by karimhbadr1** [comment]
Seems like it's something with my computer, particularly probably because it is a work computer?
I tried installing other packages and everything works perfectly, so not sure why the internet is failing on this package specifically.
I downloaded that tar files on my personal computer, got those to my work laptop, and installed from source on my work laptop, so there was a work around!
Thanks for your prompt response, appreciate it!
